src/+replab/PermutationGroup.m

```diff
@@ -241,15 +241,17 @@
         end
 
     end
-    
+
     methods(Static)
-        %% Sign
 
         function sign = sign(perm)
-            %Returns sign of a given permuatation
-
-            %perm (row vector): Vector representing a permutation (e.g. [3 2 1 4])
-            %sign (float): sign of the permutation
+        % Returns the sign of a given permutation
+        %
+        % Args:
+        %   perm (permutation): Vector representing a permutation (e.g. [3 2 1 4])
```

Comment:
We use "permutation" to describe row vectors of double type containing permutations.

Same for "integer" when a double value encodes a (smallish) integer.